### DISCUSSION TITLE
Deleting multiple screens at once/ deleting everything under "/firstRoute" in design tab

### DISCUSSION THREAD

**Discussion by hertilizer** [Budibase/budibase#4396] [Ideas]
Instead of having to manualle delete every subscreen of a route, there should be a way to delete multiple screens at once. For instance, instead of deleting "/person/:id", "/person/new/row", & "/person/", there should be a way to delete them all at once under the tab "/person" or their should be a way of selecting multiple screens and choosing to delete all.

**Comment by kasgithub8** [comment]
And copy + paste multiple screens at once!
